### PR TITLE
[1151] Add edit vacancies link

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Editing vacancies

### Changes proposed in this pull request
Add edit vacancies link to courses#index table

### Guidance to review
`https://localhost:44364/organisation/XXX/courses`

### CODE REVIEW

File: src/ui/Controllers/CourseController.cs [hunk 1]
```diff
@@ -362,6 +364,12 @@ public async Task<IActionResult> FeesPost(string providerCode, string accreditin
             return RedirectToAction("Show", new { providerCode = providerCode, accreditingProviderCode = accreditingProviderCode, courseCode });
         }
 
+        [Route("{providerCode}/course/{accreditingProviderCode=self}/{courseCode}/vacancies")]
+        public async Task<IActionResult> Vacancies(string providerCode, string accreditingProviderCode, string courseCode)
+        {
+          return frontendUrlService.RedirectToFrontend("/organisations/" + providerCode + "/courses/" + courseCode + "/vacancies");
```
Comment (timabell) [inline src/ui/Controllers/CourseController.cs]: suggest
`$"/organisations/{providerCode}/courses/{courseCode}/vacancies"`

File: src/ui/Views/Course/Index.cshtml [hunk 2]
```diff
@@ -101,7 +101,12 @@
               }
           </td>
           <td class="govuk-table__cell">@(course.IsRunning() ? course.GetApplicationStatus() : "")</td>
-          <td class="govuk-table__cell">@(course.IsRunning() ? course.GetHasVacancies() : "")</td>
+          <td class="govuk-table__cell">
+            @if(course.IsRunning()) {
```
Comment (defong) [inline src/ui/Views/Course/Index.cshtml]: @if(course.IsRunning() && course.GetHasVacancies())

Comment (benilovj) [reply src/ui/Views/Course/Index.cshtml]: when will `course.GetHasVacancies` be null?

Comment (benilovj) [reply src/ui/Views/Course/Index.cshtml]: image

Comment (defong) [reply src/ui/Views/Course/Index.cshtml]: Nevermind me I thought it was a bool

**Comment by tvararu** [comment]
@benilovj updated, tested locally with `/organisation/2ch` that I have updated on the dev db to not be opted in. (`update provider set opted_in = false where provider_code = '2CH';`):

Screenshot 2019-04-02 at 16 06 50

Almost all other providers:

Screenshot 2019-04-02 at 16 06 59

File: src/ui/Controllers/CourseController.cs (hunk 1 above)
Comment (tvararu) [reply src/ui/Controllers/CourseController.cs]: Sure but this is fine :D

**Pull request merged**